tools/run_tests/xds_k8s_test_driver/kubernetes-manifests/client.deployment.yaml

```diff
@@ -55,6 +55,8 @@ spec:
             value: "true"
           - name: GRPC_EXPERIMENTAL_ENABLE_OUTLIER_DETECTION
             value: "true"
+          - name: GRPC_EXPERIMENTAL_XDS_ENABLE_OVERRIDE_HOST
```

Comment:
This flag should be already on master.

Reply:
Good catch. I meant to merge master before pushing, but apparently forgot